src/gui/editors/AutomationEditor.cpp

```diff
@@ -180,6 +182,10 @@ AutomationEditor::AutomationEditor() :
 	{
 		s_toolDrawOut = new QPixmap(embed::getIconPixmap("edit_draw_outvalue"));
 	}
+	if (s_toolEditTangents == nullptr)
+	{
+		s_toolEditTangents = new QPixmap(embed::getIconPixmap("edit_tangent"));
```

Comment:
I think this line introduces a memory leak.

@sakertooth recently opened a PR (#6879) which will change each instance of `new QPixmap` throughout the codebase to allocate on the stack instead.

Reply:
That one will depend on which PR will be merged first. If his, then this one will have another merge conflict and that will have to be fixed anyways, and vice-versa. So I think it's safe to leave it as it is for now.

Reply:
I'm predicting that this will get merged first, so I will make note of this :+1:.